### PR TITLE
test: add cypress sanity check

### PR DESCRIPTION
**Pull request by marcelltoth**
Part of stoplightio/elements#659

Adds a simple cypress test to this repo. It loads the Zoom API (API component) and checks if it loads.

The test is not intended to be complete. My goal here is to prove that the cypress setup works and that I can run this test against an elements-PR. (This second part is to come in a separate PR against elements, but needs this merged first.)

*Commands added*:
- `yarn serve-prod` Launches a simple HTTP server serving the output of an `ng build --prod` (from folder `./dist/angular`) on `http://localhost:4200`.
- `yarn e2e:open` Launches a Cypress window, as in `platform-internal`
- `yarn e2e:execute` Runs the Cypress test suite. This expects that the HTTP server is up and serving the site on *localhost*
port 4200
- `yarn e2e:run` Launches the server and runs the Cypress test suite. Think of it as `yarn serve-prod && yarn e2e:execute; kill (pid of HTTP server)`

### CODE REVIEW

**Comment by marcelltoth** [comment]
Note: We might also want to add a check in this repo that runs this test against new PRs to make sure we don't break this example and the test.

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mallachari) [inline cypress.json:1]: I guess it's mandatory or will be needed later on?

File: package.json [hunk 2]
```diff
@@ -13,7 +13,11 @@
     "start": "ng serve",
     "build": "ng build --prod --base-href '/elements-starter-angular'",
     "postinstall": "ngcc --tsconfig './tsconfig.app.json'",
-    "lint": "tslint 'src/**/*.ts'"
+    "lint": "tslint 'src/**/*.ts'",
+    "serve-prod": "angular-http-server --path ./dist/angular -p 4200",
+    "e2e:execute": "cypress run",
+    "e2e:open": "cypress open",
+    "e2e:run": "start-server-and-test serve-prod http://localhost:4200 e2e:execute"
```
Comment (mallachari) [inline package.json:20]: What's the benefit of using `start-server-and-test`? Shouldn't it run just with `serve-prod`?

**Review by mallachari — APPROVED** [review]
Builds and runs well. 
There are few skeleton files but I assume we'll fill them later.

File: cypress.json (hunk 1 above)
Comment (marcelltoth) [reply cypress.json:1]: It was part of the template that cypress installs automatically, haven't checked if it's really necessary but it'll likely be needed later, I'd keep it.

File: package.json (hunk 2 above)
Comment (marcelltoth) [reply package.json:20]: `start-server-and-test` is a simple script that orchestrates the following:
1. Starts `yarn serve-prod`
2. Waits until it is up and running by pinging `http://localhost:4200`
3. When it is, it executes `yarn e2e:execute`
4. When `e2e:execute` quits, it tears down the server.

**Pull request merged**